Client/mods/deathmatch/logic/luadefs/CLuaAudioDefs.cpp

```diff
@@ -450,6 +455,61 @@ int CLuaAudioDefs::GetSoundBufferLength(lua_State* luaVM)
     return 1;
 }
 
+int CLuaAudioDefs::SetSoundLooped(lua_State* luaVM)
+{
+    CClientSound*    pSound;
+    bool             bLoop;
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData(pSound);
+    argStream.ReadBool(bLoop);
+
+    if (!argStream.HasErrors())
+    {
+        if (pSound)
+        {
+            lua_pushboolean(luaVM, pSound->SetLooped(bLoop));
+            return 1;
+        }
+        else
+        {
+            m_pScriptDebugging->LogBadPointer(luaVM, "sound", 1);
+            lua_pushboolean(luaVM, false);
+        return 1;
+        }
```

Comment:
no need to do `if (pSound)`, I think pSound is guaranteed to exist because of ReadUserData